docker/alpine/Dockerfile

```diff
@@ -22,17 +22,23 @@ RUN \
 FROM alpine:latest
 LABEL maintainer "John Eikenberry <jae@zhar.net>"
 
-# This is the release of https://github.com/hashicorp/docker-base to pull in order
-# to provide HashiCorp-built versions of basic utilities like dumb-init and gosu.
+# This is the release of https://github.com/hashicorp/docker-base to pull in
+# order to provide HashiCorp-built versions of basic utilities like dumb-init
+# and gosu.
```

Comment:
From the changes below, it seems like `gosu` has been removed. If i understood correctly, then maybe worth removing `and gosu` from comment?

Reply:
Good catch. I'll remove that. Thanks.